src/components/Footer/Footer.react.js

```diff
@@ -116,19 +123,36 @@ const Footer = () => {
               <li>
                 <Link to="/support">Support</Link>
               </li>
+              {isMobileView() && (
```

Comment:
Instead of calling `isMobileView()` multiple times, call it only once before return statement and use the value elsewhere.

Reply:
Resolved this comment